src/referrals.cpp

```diff
@@ -31,6 +31,34 @@ MaybeReferral ReferralsViewCache::GetReferral(const Address& address) const
     return {};
 }
 
+MaybeReferral ReferralsViewCache::GetReferral(const std::string& alias, bool normalize_alias) const
+{
+    auto maybe_normalized = alias;
+
+    if (normalize_alias) {
+        NormalizeAlias(maybe_normalized);
+    }
+
+    if (maybe_normalized.size() == 0) {
+        return {};
+    }
+
+    {
+        LOCK(m_cs_cache);
+        if (alias_index.count(maybe_normalized)) {
```

Comment:
use alias_index.find here instead of count to get the iterator. This is faster because both count and the [ ] have to do the same lookup.